src/Root.js

```diff
@@ -46,91 +59,6 @@ const AccessError = message => {
   return instance;
 };
 
-Relay.injectNetworkLayer(
-  new RelayNetworkLayer([
-    urlMiddleware({
-      url: req => urlJoin(API, 'graphql'),
-    }),
-    retryMiddleware({
-      fetchTimeout: 15000,
-      forceRetry: (cb, delay) => {
-        window.forceRelayRetry = cb;
-        console.log(
-          `call \`forceRelayRetry()\` for immediately retry! Or wait ${delay} ms.`
-        );
-      },
-      retryDelays: attempt => ((attempt + 4) ** 2) * 100,
-      statusCodes: retryStatusCodes,
-    }),
-    // Add hash id to request
-    next => req => {
```

Comment:
Abstracting this into a recompose handler, to be passed into the Relay network layer after component mount.